srcpkgs/sway/template

```diff
@@ -1,19 +1,23 @@
 # Template file for 'sway'
 pkgname=sway
-version=0.15.2
+version=1.0
 revision=1
-build_style=cmake
-configure_args="-DCMAKE_INSTALL_SYSCONFDIR=/etc"
-hostmakedepends="asciidoc pkg-config wayland-devel"
-makedepends="gdk-pixbuf-devel json-c-devel libcap-devel pam-devel pango-devel
- wlc-devel"
+_version=$version-alpha.6
+wrksrc=$pkgname-$_version
+build_style=meson
+configure_args="-Dsway-version=$_version"
+hostmakedepends="pkg-config wayland-devel wayland-protocols scdoc"
+makedepends="wlroots-devel libinput-devel libcap-devel pcre-devel
+ json-c-devel pango-devel cairo-devel gdk-pixbuf-devel pam-devel
+ libxkbcommon-devel pixman-devel"
+depends="xorg-server-xwayland"
 conf_files="/etc/sway/config /etc/pam.d/swaylock"
-short_desc="An i3-compatible window manager for Wayland"
+short_desc="i3-compatible Wayland compositor"
```

Comment:
This generated a lint error because i3 is lowercase. Perhaps "Tiling Wayland compositor compatible with i3"?

Reply:
Fair enough.